src/integrations/admin/deactivated-premium-integration.php

```diff
@@ -112,9 +112,8 @@ function dismiss_premium_deactivated_notice(){
                     'action': 'dismiss_premium_deactivated_notice',
                     };
 
-                    jQuery.post( ajaxurl, data, function( response ) {
-                        jQuery( '#yoast-premium-deactivated-notice' ).hide();
-                    });
+                    jQuery( '#yoast-premium-deactivated-notice' ).hide();
+                    jQuery.post( ajaxurl, data, function( response ) {});
```

Comment:
Why change this to hide right away? Maybe add a tech choice about it?
I suppose we don't care about whether it succeeded or not

Reply:
Because it's already hidden right away for some reason in the old notices (that is, before the ajax call is completed)